scripts/hail_batch/vep/run_vep.py

```diff
@@ -25,6 +25,7 @@ def main(mt: str):
     mt = mt.filter_rows(hl.len(mt.alleles) == 2)
     mt = mt.filter_rows(mt.alleles[1] != '*')
     vep = hl.vep(mt, config='file:///vep_data/vep-gcloud.json')
+    vep = vep.rows()
```

Comment:
```suggestion
    ht = mt.rows()
    # Remove ref-blocks.
    ht = ht.filter(ht.alleles[1] != '*')
    vep = hl.vep(ht, config='file:///vep_data/vep-gcloud.json')
```

Reply:
removing reference blocks would be this line: `mt.filter_rows(hl.len(mt.alleles) == 2)`, so I think that line should stay and the comment should be moved to the relevant place

Reply:
Yes, you're absolutely right, sorry for mixing that up. Instead of filtering for `'*'`, you could also use `hl.is_snp`.

Reply:
Interesting, I didn't know about that one. So something like
`ht.filter_rows(hl.is_snp(ht.alleles[0], ht.alleles[1]))`? Wouldn't it also be good to have annotation for transversions/insertions/deletions, etc here as well? So maybe something like `ht.filter_rows(hl.is_star(ht.alleles[0], ht.alleles[1]), keep = False)`?

Reply:
Maybe we can just delete `ht = ht.filter(ht.alleles[1] != '*')` completely then?

Reply:
Unfortunately VEP doesn't work with starred alleles: https://discuss.hail.is/t/vep-output-variant-not-found-in-original-variants/1148